src/components/PasswordRequirement/PasswordRequirement.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import "./scss/PasswordRequirement.css";
+export const passwordRequirements = (password) => {
+  const lengthCheck = /.{8,}/.test(password);
+  const upperCaseCheck = /[A-Z]/.test(password);
+  const lowerCaseCheck = /[a-z]/.test(password);
```

Comment:
Make this as utility function which has switch case statement, return the error from that function 
Exxample: 
passwordRequirement()=> returns "Minimum 8 characters" and errorState
Based on that handle in JSX